utils/convert.js

```diff
@@ -1,22 +1,12 @@
-const fs = require('fs');
-const path = require('path');
+import fs from 'fs';
```

Comment:
On Node 18 I'm getting this error:
```
(node:79693) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/Users/dpagurek/Documents/Projects/p5.js-2/utils/convert.js:1
import fs from 'fs';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

@limzykenneth how do you feel about bumping the minimum nodejs version for 2.0? This syntax works on node 20

Reply:
We don't really have a standard way of defining the minimum supported node.js but yes, happy with node 20 minimum. Will need to check CI is using node 20.

Reply:
oh looks like they already use 20, perfect! I'll change the referenced node version in the contributor docs to match
https://github.com/processing/p5.js/blob/7982e2f53e8d07836b4dfdf7cb175cfeb56555a9/.github/workflows/ci-test.yml#L19-L20